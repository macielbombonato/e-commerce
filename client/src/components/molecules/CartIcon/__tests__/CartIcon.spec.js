import createTestContext from 'react-cosmos-test/enzyme';
import toJson from 'enzyme-to-json';

import fixture from '../CartIcon.fixture';

const { mount, getWrapper } = createTestContext({fixture});

beforeEach(mount);

describe('#CartIcon', () => {
  it('renders component', () =>
    expect(toJson(getWrapper())).toMatchSnapshot()
  );
});
