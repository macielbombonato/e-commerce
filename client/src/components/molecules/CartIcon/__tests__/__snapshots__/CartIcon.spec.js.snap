// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#CartIcon renders component 1`] = `
.glamor-2,
[data-glamor-2] {
  color: black;
  font-size: 0.5rem;
  height: 1.5rem;
  position: relative;
  width: 1.5rem;
}

.glamor-0,
[data-glamor-0] {
  width: 100%;
  height: 100%;
}

.glamor-1,
[data-glamor-1] {
  position: absolute;
  top: 0.6rem;
  background: red;
  height: 14px;
  width: 14px;
  line-height: 15px;
  text-align: center;
  right: 0;
}

<CartIcon
  className="custom-class"
  quantity={5}
>
  <Link
    replace={false}
    to="/checkout"
  >
    <a
      href="/checkout"
      onClick={[Function]}
    >
      <glamorous(div)
        className="custom-class"
      >
        <div
          className="glamor-2 custom-class"
        >
          <glamorous(img)
            src="cart.svg"
          >
            <img
              className="glamor-0"
              src="cart.svg"
            />
          </glamorous(img)>
          <glamorous(div)
            className="rounded-circle"
          >
            <div
              className="glamor-1 rounded-circle"
            >
              5
            </div>
          </glamorous(div)>
        </div>
      </glamorous(div)>
    </a>
  </Link>
</CartIcon>
`;
